src/relay/backend/contrib/cutlass/codegen.cc

```diff
@@ -43,6 +43,16 @@ namespace cutlass {
 
 namespace {
 
+/*! \brief Return the "cutlass" Target instance to use to guide compilation. */
+Target GetCutlassTarget() {
+  Target target = Target::Current(/*allow_not_defined=*/true);
+  if (!target.defined() || target->kind->name != "cutlass") {
+    // Use the default CUTLASS compilation options.
+    target = Target("cutlass");
```

Comment:
how come target->kind->name != "cutlass" makes this happen? seems like it should icheck-fail?

Reply:
I'll beef up the comment. This is so folks who just want to use the default CUTLASS compilation options (which are all very sensible) don't need to futz with multi targets